.vscode/settings.json

```diff
@@ -29,5 +28,6 @@
   ],
   "cSpell.ignoreWords": ["TESTTEXTBLOBNAME", "TESTWASMNAME", "yxxx"],
   "eslint.runtime": "node",
+  "eslint.lintTask.options": "--cache --cache-strategy content",
```

Comment:
This _should_ enable using `.eslintcache` for checks run by the plugin, but I can't figure out where it actually generates the `.eslintcache` file.

Reply:
I don't think it makes any difference. See https://github.com/microsoft/vscode-eslint/issues/1323

Reply:
good shout, thanks. I'll remove it.